CHANGELOG.md

```diff
@@ -171,6 +171,7 @@
 * Fixed an issue, which prevented the ability to change `UserPuckCourseView.puckColor` when changing system-wide appearance. ([#3306](https://github.com/mapbox/mapbox-navigation-ios/pull/3306))
 * `FeedbackViewController` now can be customized to show only feedback types specific to passive navigation. Pass `FeedbackViewControllerType` to the constructor of the feedback view controller to configure the available feedback categories. ([#3323](https://github.com/mapbox/mapbox-navigation-ios/pull/3323))
 * Renamed `FeedbackType` to `ActiveNavigationFeedbackType` and `EventsManagerDataSource` to `ActiveNavigationEventsManagerDataSource`. ([#3327](https://github.com/mapbox/mapbox-navigation-ios/pull/3327))
+* Feedback catgories and subcategories for active navigation were changed. For the updated list of supported categories see file `ActiveNavigationFeedbackType.swift`. ([#3339]((https://github.com/mapbox/mapbox-navigation-ios/pull/3339))
```

Comment:
Type in catgories.

Reply:
It would be helpful to name at least some of the symbols that got renamed, so that jazzy can link to their documentation. It won’t be able to link to a bare file name like this.